examples/python/compilation_example.py

```diff
@@ -233,7 +233,7 @@ def cost(circ):
 q = [Qubit("q", i) for i in range(5)]
 c = Bit("c")
 circ.add_bit(c)
-circ.Measure(q[3], c)
+# circ.Measure(q[3], c)
```

Comment:
It looks like we added a no mid circuit measurement predicate which was not here before in pytket-qiskit?

Reply:
Where is the error this is fixing? It looks like we already wrap `PauliSimp().apply()` in a try/catch block...